SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbarCommands.cpp

```diff
@@ -10,9 +10,9 @@ void FSpatialGDKEditorToolbarCommands::RegisterCommands()
 	UI_COMMAND(CreateSpatialGDKSchemaFull, "Schema (Full Scan)", "Creates SpatialOS Unreal GDK schema for all assets.", EUserInterfaceActionType::Button, FInputGesture());
 	UI_COMMAND(DeleteSchemaDatabase, "Delete schema database", "Deletes the scheme database file", EUserInterfaceActionType::Button, FInputGesture());
 	UI_COMMAND(CreateSpatialGDKSnapshot, "Snapshot", "Creates SpatialOS Unreal GDK snapshot.", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartNoAutomaticConnection, "Start", "Not Automatic Connect to service", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartLocalSpatialDeployment, "Local", "Connect to local deployment service", EUserInterfaceActionType::Button, FInputGesture());
-	UI_COMMAND(StartCloudSpatialDeployment, "Cloud", "Connect to cloud SpatialOS service", EUserInterfaceActionType::Button, FInputGesture());
+	UI_COMMAND(StartNoAutomaticConnection, "None", "Doesn't automatically connect to SpatialOS", EUserInterfaceActionType::Button, FInputGesture());
```

Comment:
I believe docs communication standards are to not use shortened versions of words.
```suggestion
	UI_COMMAND(StartNoAutomaticConnection, "None", "Does not automatically connect to SpatialOS", EUserInterfaceActionType::Button, FInputGesture());
```

Reply:
Since the others mention `deployment` should we be using `SpatialOS deployment` here or perhaps just `deployment`. 
Oh god I sound like @anne-edwards

Reply:
@joshuahuburn it's a slippery slope - before you know it, you'll be a full-blown tech writer 😬 
(Re. shortened words... personally I like them, because they're less stuffy/formal, but I don't really mind either way.)